research/quantifying-data-leakage/README.md

```diff
@@ -74,41 +74,48 @@ training image and iteration. Therefore, the batch size is 1.
 
 We modify the client's config file to use the downloaded data and use the 
 prepared dataset list for client 9. 
+Furthermore, we need to modify the gradient inversion config file to use 
+the downloaded prior image (part of DeepInversion package).
+
+This can be done by using the provided template configure files in 
+[./job_configs/app_template](./job_configs/app_template).
 ```
-python3 -m nvflare_gradinv.utils.modify_config --config_file \
-./job_configs/cxr_1client/client/config/config_fed_client.json \
---data_root=${DATA_ROOT} \
---dataset_json=${PWD}/data/data_200val+rest_client9.json
+python3 -m nvflare_gradinv.utils.create_job_config --app_folder ./job_configs/app_template \
+--job_config ./job_configs/cxr_1client --n_clients 1 --num_rounds 3 \
+--data_root=${DATA_ROOT} --dataset_json=${PWD}/data/data_200val+rest_client \
+--prior_file=${PWD}/DeepInversion/FLGradientInversion/prior/prior_1.jpg
```

Comment:
just my preference, can we move the "output argument" to the very end?

Like:

``python3 -m nvflare_gradinv.utils.create_job_config --app_folder ./job_configs/app_template \
--n_clients 1 --num_rounds 3 \
--data_root=${DATA_ROOT} --dataset_json=${PWD}/data/data_200val+rest_client \
--prior_file=${PWD}/DeepInversion/FLGradientInversion/prior/prior_1.jpg
--job_config ./job_configs/cxr_1client 
``

Also, I am wondering can "job_config" to be named something else so people know it is a output directory from the argument name?

Reply:
Sure. Changed it to end and called it "output".